js/ui/marker.js

```diff
@@ -13,6 +13,7 @@ var Point = require('point-geometry');
  * @param {HTMLElement=} element DOM element to use as a marker (creates a div element by default)
  * @param {Object=} options
  * @param {PointLike=} options.offset The offset in pixels as a [`PointLike`](#PointLike) object to apply relative to the element's top left corner. Negatives indicate left and up.
+ * @param {boolean=} [options.perspective=false] Whether the marker should scale along with the map geometry.
```

Comment:
Or perhaps `pitchScale: "map"`, for consistency with the [`circle-pitch-scale: map`](https://www.mapbox.com/mapbox-gl-style-spec/#paint-circle-pitch-scale) style property?